### PR TITLE
Add Arrowhead wrappers

### PR DESCRIPTION
**Pull request by jara001**

### CODE REVIEW

**Review by wentasah — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,26 @@
-# f1tenth-scoreapp
+# F1Tenth Scoreapp
+
+**F1Tenth Scoreapp** is a complete solution for measuring lap time on a racing track, which was developed for [International F1/10 Autonomous Racing Competition](https://f1tenth.org/).
 
 This project consists of several parts:
 1. [frontend](./frontend/) (client-side React.js app)
 2. [backend](./backend/) (Go server providing REST and WebSocket APIs)
-3. [optical barrier code and design](./barrier/) (KiCAD, C, Czech)
+3. [backend-arrowhead](./backend-arrowhead/) (Arrowhead Compliant overlay for launching the backend)
```
[the remark this answers is not in the document]
Comment (jara001) [reply README.md]: Moved to `./arrowhead`. Used `wrapper` in readme even though I still believe that it was properly used. (Not everything is ROS (: ).

File: barrier/optic_barrier_sw/CMakeLists.txt [hunk 2]
```diff
@@ -2,6 +2,66 @@ cmake_minimum_required(VERSION 3.1.0)
...
+    set(GIT_AVAILABLE false)
+endif()
+
+# Preset variables
+set(GIT_IS_DIRTY false)
+
+if(GIT_AVAILABLE)
+    # Git describe
+    execute_process(
+        COMMAND git describe --always --tags
```
[the remark this answers is not in the document]
Comment (jara001) [reply barrier/optic_barrier_sw/CMakeLists.txt:27]: It is not copied, some parts are probably inspired somewhere else. During the development I believe that I have used most of them, but not now.

^^ describe, describe=40 is not the same. The goal was to create a unique commit name that fits on the screen. (Also, dirty can be shown using something else, thus it is not used in describe.)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,26 @@
...
+4. [optical barrier code and design](./barrier/) (KiCAD, C, Czech)
+5. [barrier-arrowhead](./barrier-arrowhead/) (Arrowhead Compliant overlay for the optic barrier)
 
 ![Optical barrier based on Raspberry Pi Zero](doc/barrier.jpg)
+
+
+## License
+
+F1tenth Scoreapp is released under the **GPLv3** license.
```
[the remark this answers is not in the document]
Comment (jara001) [reply README.md:17]: I will change it to TODO and open up an issue here.

File: barrier/optic_barrier_sw/main.c [hunk 4]
```diff
@@ -179,34 +210,84 @@ void update_display(enum screen screen)
...
+        gettimeofday(&start2, NULL);
 
     gettimeofday(&now, NULL);
 
-    if (usec_between(&start, &now) > 5*1000000) {
+    if (usec_between(&start2, &now) > 5*1000000) {
         update_display(EMPTY);
         System_Exit();
-        system("sudo shutdown -h now");
+        system("pkill websocat"); // FIXME: This is currently only way how I can force quit on websocat.
```
[the remark this answers is not in the document]
Comment (jara001) [reply barrier/optic_barrier_sw/main.c:290]: I have moved this to #8 as it requires further attention and as it is not required right now.

**Pull request merged**